openapi/components/requestBodies/storefront/PostKycLivenessSession.yaml

```
@@ -0,0 +1,14 @@
+content:
+  application/json:
+    schema:
+      type: object
+      required:
+        - kycRequestId
+      properties:
+        kycRequestId:
+          type: string
+          description: ID of the KYC request that the liveness session is associated with.
+          maxLength: 50
+          example: kyc_req_0YV7JMJ3DBCGRBR7K9D4HVGPP5
+description: KYC Liveness Session.
```

nitpick: capitalization 
```suggestion
description: KYC liveness session.
```